setup.sh

```diff
@@ -1,10 +1,29 @@
 #!/bin/bash
 set -e
 
-if [[ $DISABLE_VCS != "true" ]]
-then
+# Display warning message
+function warn() {
+    message=$1
+    echo -e "\e[1;33m${message}\e[0m"
+}
+
+# Dependency installation for websites
+function installWebsiteDependencies() {
+    # Install dependencies for docs website
+    echo "Installing docs website Python dependencies..."
+    pip3 install -Ur websites/docs/docs/requirements.txt --upgrade-strategy eager
```

Comment:
I don't think an eager upgrade strategy is needed. Upgrade strategies are for dependencies of the upgraded packages, not the upgraded packages themselves. What do you think?

From the output of `pip install --help`
```
 --upgrade-strategy <upgrade_strategy>
                              Determines how dependency upgrading should be handled [default: only-if-needed].
                              "eager" - dependencies are upgraded regardless of whether the currently installed version satisfies the requirements of the upgraded package(s).
                              "only-if-needed" -  are upgraded only when they do not satisfy the requirements of the upgraded package(s).
```